scripts/irods/test/test_dynamic_peps.py

```diff
@@ -533,7 +535,7 @@ def test_if_StructFileExtAndRegInp_is_exposed__issue_7413(self):
                 self.user.assert_icommand(['imkdir', TEST_COLLECTION_NAME + "_backup"], 'STDOUT')
 
                 # Should trigger PEP
-                self.user.assert_icommand(['ibun', '-x', TEST_TARFILE_NAME, TEST_COLLECTION_NAME + "_backup"])
+                self.user.assert_icommand(['ibun', '-x', '-f', TEST_TARFILE_NAME, TEST_COLLECTION_NAME + "_backup"])
```

Comment:
Just to confirm, the `-f` is to exercise the `forceFlag`, right?

Reply:
Yes!

Reply:
Okey dokey. Resolving